test/package.json

```diff
@@ -5,7 +5,7 @@
     "TODO-STENCIL-389": "echo Remove the exit call below",
     "analysis": "exit 0 && node ./.scripts/analysis.js",
     "analysis.build-and-analyze": "npm run build && npm run analysis",
-    "build": "npm run build.browser-compile && npm run build.hello-world && npm run build.hello-vdom && npm run build.todo && npm run build.end-to-end && npm run build.ionic && npm run build.docs-json && npm run build.docs-readme",
+    "build": "npm run build.browser-compile && npm run build.hello-world && npm run build.hello-vdom && npm run build.todo && npm run build.end-to-end && npm run build.ionic",
```

Comment:
Does this exclude this from analysis testing?

Reply:
yeah, formerly these were built as part of the analysis tests but now they're only built on their own

the motivation for making the change overall was that the analysis tests are sort of in this murky area and it would be better if we know for sure that something is valuable and important to pull it out into its own separate thing